node/test/tasklib.ts

```diff
@@ -803,6 +803,20 @@ describe('Test vsts-task-lib', function () {
             assert.equal(tc.toString(), '##vso[some.cmd]a message');
             done();
         })
+        it('handles boolean properties', function (done) {
+            this.timeout(1000);
+
+            var tc = new tcm.TaskCommand('some.cmd', { foo: false }, 'a message');
```

Comment:
Agree with that. Since the `string` this is at compile time and task can actually send command like mentioned in UT at run time. This check skips that property and doesn't send, which shouldn't be the case.